### ISSUE TITLE
Group by semver tag based on chore(release) commits

### ISSUE THREAD

**Issue by mmorainville** [conventional-changelog/conventional-changelog#205]
Hi,

I believe it's usually a good practice to to use this kind of commit message when making a new release:
```
chore(release): v0.4.2
```
And then tag this commit with `v0.4.2`. So it would be great that conventional-changelog recognize this kind of commit message to group the items by tag.

In its current state, we have:

> The above generates a changelog based on commits since the last semver tag that match the pattern of a "Feature", "Fix", "Performance Improvement" or "Breaking Changes".

So that means that we must put the tag on a `feat(release): v0.4.2` commit for exemple. But it's not what `feat` should be used for.

So I suggest to also match the pattern of a chore(release) commit message to generate a changelog based on semver tag.

Regards,
Mathieu

**Comment by stevemao** [comment]
> The above generates a changelog based on commits since the last semver tag that match the pattern of a "Feature", "Fix", "Performance Improvement" or "Breaking Changes".

It just means that other verbs including `chore` will be ignored by default. So `chore(release): v0.4.2` will not be in your CHANGELOG.

`chore(release): v0.4.2` is what we recommend.

**Comment by mmorainville** [comment]
Hi @stevemao,

Thanks for your input, but the thing is, when I run `standard-changelog --first-release` or `standard-changelog -r 10` the generated CHANGELOG.md does list the correct versions as it looks after tagged commits with feat, fix, perf, etc. but not chore.

But my problem is that I have let's say the following:
- chore(release): v0.4.3 (tagged with v0.4.3)
- fix: some fix
- feat: some new feat
- chore(release): v0.4.2 (tagged with v0.4.2)
- feat: some feat

But then, it seems standard-changelog just ignores the 0.4.2 and 0.4.3 versions because the tag is on a `chore` commit and not a `feat` commit.

If I have this:
- chore(release): v0.4.3
- fix: some fix (tagged with v0.4.3)
- feat: some new feat
- chore(release): v0.4.2
- feat: some feat (tagged with v0.4.2)

It seems to work correctly, but the tag is not properly put on the chore(release) commit.
So the parser should recognize `chore` commits for versions recognition but not include it in the changelog (as it is now).

Regards,
M.

**Comment by stevemao** [comment]
> as it looks after tagged commits with feat, fix, perf, etc. but not chore.

This is not correct. It looks after all tags regardless of the commit message. What version are you using? Could you use the [verbose option](https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-cli/cli.js#L42) to see what's been printed?

**Comment by dangreen** [comment]
Closing as stale because the requested version, verbose output, and reproducible repository have not been provided since 2017. Please open a new issue with a current minimal reproduction if this is still a problem.
